desktop/nested-virtualization.md

```diff
@@ -0,0 +1,50 @@
+---
+description: Instructions on how to enable nested virtualization
+keywords: nested virtualization, Docker Desktop
+title: Enable nested virtualization
+---
+>Note: 
+>
+>This is work in progress. The steps haven’t been validated yet in our test environment.
+{: .important}
+
+
+In general, Docker recommends running Docker Desktop natively on either Mac, Linux, or Windows. However, Docker Desktop can run inside a virtual machine provided the virtual desktop is properly configured. 
```

Comment:
here, we either say virtual machine or virtual desktop on both instances:
_However, Docker Desktop can run inside a **virtual machine** provided the **virtual desktop** is properly configured._